src/runtime_src/core/common/device.h

```diff
@@ -34,6 +34,8 @@
 
 namespace xrt_core {
 
+using DeviceCollection = std::vector<std::shared_ptr<xrt_core::device>>;
```

Comment:
Lets drop camel case and consistently use _
At least in the classes that are shared across the system.
I don't care about the tools/common sources

Reply:
Made changes as suggested.